src/test/repository.test.ts

```diff
@@ -1938,8 +1938,6 @@ test('setMode can switch from streaming to polling mode', async (t) => {
   t.is(repo.getMode(), 'polling');
   t.true(eventSource.closed);
 
-  await repo.fetch();
```

Comment:
the test will now work without manual fetch trigger and based on the mode change alone